state/watcher/watcher.go

```diff
@@ -103,6 +103,10 @@ type event struct {
 	revno int64
 }
 
+// Period is the delay between each sync.
+// It must not be changed when any watchers are active.
```

Comment:
if there was a race, wouldn't it imply that a test *is* changing Period while a watcher is running? shouldn't we fix that?